spec/models/abuse_report_spec.rb

```diff
@@ -5,10 +5,6 @@
     it "is valid" do
       expect(build(:abuse_report)).to be_valid
     end
-
```

Comment:
Instead of deleting this test, should it be switched to test that an abuse report "is not valid without an email"?